.cli/init.js

```diff
@@ -89,10 +143,26 @@ const main = () => {
   ]);
 
   // Update info.yml file.
-  const infoFilePath = path.join(__dirname, `../${machineName}.info.yml`);
-  const info = yaml.safeLoad(fs.readFileSync(infoFilePath, 'utf8'));
-  info.name = name;
-  fs.writeFileSync(infoFilePath, yaml.safeDump(info));
+  applyToYmlFile(
+    path.join(__dirname, `../${machineName}.info.yml`),
+    (info) => ({
+      ...info,
+      name: machineName,
+      libraries: info.libraries.map(strReplaceEmulsify(machineName)),
```

Comment:
I am thinking we should remove this line as part of this PR: https://github.com/emulsify-ds/emulsify-drupal/blob/master/emulsify.info.yml#L11

Reply:
It is removed. I am loading the file, parsing it into a JS object, modifying properties on the object, converting it back to yml, and then writing it to the file. Since comments are not part of that object, they are stripped.

Now that I bring that up, I figured it was ideal to strip out those inactive comments (as people can always refer to the canonical file), but what do you think?

If we want to preserve the comments, I'd either need to update the AST in the js-yaml library (or find one that preserves comments), or load the file as a string and do string manipulation.

Reply:
Oh neat, here's a different library that does support comments: https://www.npmjs.com/package/yaml

Either way, I defer to you as to whether or not we should keep comments.

Reply:
@patrickocoffeyo No need to keep the comments :+1: , you are good to go here, I wanted it gone.